wpe/src/main/cpp/browser/messagepump.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <glib.h>
+#include <memory>
+#include <vector>
+
+struct ALooper;
+
+class MessagePump
+{
+public:
+    MessagePump();
+
+    void quit();
+
+    void invoke(void (* callback)(void*), void* callbackData, void (* destroy)(void*));
+
+    void onCollectEventsLooperCallback(int fd, int events);
+    void onDispatchLooperCallback();
+
+private:
+
+    void prepare();
+    void dispatch();
+
+    ALooper* m_looper = nullptr;
+    int m_dispatchFd = 0;
+
+    GMainContext *m_context;
```

Comment:
Maybe initialize it to nullptr too, to follow the other initializations.

Reply:
done